build.psm1

```diff
@@ -1533,7 +1533,7 @@ function Start-PSBootstrap {
                 Start-NativeExecution { brew install $Deps } -IgnoreExitcode
 
                 # Install patched version of curl
-                Start-NativeExecution { brew install curl --with-openssl } -IgnoreExitcode
+                Start-NativeExecution { brew install curl --with-openssl --with-gssapi } -IgnoreExitcode
```

Comment:
We should update the PowerShell brew recipe as well.
https://github.com/caskroom/homebrew-cask/blob/master/Casks/powershell.rb#L33

Reply:
opened caskroom/homebrew-cask#41673

Reply:
#5635